auth0/resource_auth0_guardian.go

```diff
@@ -262,15 +260,12 @@ func createGuardian(ctx context.Context, d *schema.ResourceData, m interface{})
 func readGuardian(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
 
-	multiFactorPolicies, err := api.Guardian.MultiFactor.Policy()
+	flattenedPolicy, err := flattenMultiFactorPolicy(api)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	result := multierror.Append(d.Set("policy", "never"))
-	if len(*multiFactorPolicies) > 0 {
-		result = multierror.Append(result, d.Set("policy", (*multiFactorPolicies)[0]))
-	}
+	result := multierror.Append(d.Set("policy", flattenedPolicy))
```

Comment:
Extracting this logic somewhere else so it's easier to read through the readGuardian steps.